components/ui/input.tsx

```diff
@@ -99,6 +99,8 @@ const Input = React.forwardRef(
     const prefixPresent = typeof renderPrefix === "function";
     const suffixPresent = typeof renderSuffix === "function";
 
+    restProps.value = restProps.value ? restProps.value : "";
```

Comment:
The problem was that the dynamic fields were given `undefined` value instead of an empty string. We can't initialise those values to empty string because we don't know how many of them will be in the form since they are added dynamically. Setting the value to an empty string for those field manually got rid of the warning, but I'm not sure that's the right way 🤔 @ridjis